src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+//The transactions should all be equal EXCEPT for the Signers field
+function validateAllSignedTransactionsAreEqual( transactions : Array<JsonObject> ) {
+  const tx = transactions[0];
+  const _txSignersTemp = tx.Signers;
+  delete tx.Signers
+  
+  transactions.forEach( _tx => {
+      const _txSignersTemp = _tx.Signers
+      delete _tx.Signers;
```

Comment:
there's no reason to delete this since you are re-assigning it later. using `delete` is weird

Reply:
ah nevermind i see why you're doing this. but there's a better way - see my suggestion below!